site-pages/cart/cart.css

```diff
@@ -0,0 +1,113 @@
+.cart-container{
+    display: flex;
+    margin-top: 30px;
+}
+.cart-cards-container{
+    width: 75%;
+    display: flex;
+    flex-direction: column;
+    gap: 30px;
+    align-items: center;
+    justify-content: center;
+    margin-bottom: 2.5rem;
+}
+
+.bill{
+    display: flex;
+    float: right;
+    width: 25%;
+    height: 100%;
+    box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
+    padding: var(--content-padding);
+    margin-top: 30px;
+    margin-right: 20px;
+    justify-content: center;
+    align-items: center;
+}
+.bill-container{
+    display: flex;
+    flex-direction: column;
+
+}
+
+.cart-head{
+    font-size: 500;
+}
+.hori-img{
+    padding: 0.9375rem;
+    width: 15.625rem;
+    height: 16.875rem;
+}
+.discount{
+    text-decoration: line-through;
+    color: gray;
+}
+.discount-per{
+    color: var(--good-color);
+}
+.cart-card{
+    max-width: 90%;
+}
+.main-info-cont{
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    margin: 0.9375rem 0.9375rem;
+}
+.quantity-cont{
+    display: flex;
+    flex-direction: row;
+    gap: 5px;
+    align-items: center;
+    justify-content: center;
+    margin-left: 0px;
+    padding-left: 0px;
+}
+.quantity-head{
+    margin-right: 0px;
+    padding-right: 0px;
+}
+.quantity-btn{
+    background-color: transparent;
+    border: 1px solid var(--primary-color);
+    border-radius: 100%
+}
+.inc:hover{
+    background-color: var(--good-color);
+}
+.dec:hover{
+    background-color: var(--warning-color);
+}
+.quantity-btn:hover{
+    cursor: pointer;;
+}
+.product-quant{
+    border: 1px solid var(--primary-color);
+    border-radius: var(--border-radius);
+    padding: 0px 5px;
+}
+.fa-plus, .fa-minus{
+    color: black;
+}
+.cart-btn-cont{
+    display: flex;
+    flex-direction: column;
+}
+.rem-btn, .read-btn{
+    padding: 5px;
+}
+
+.price,
+.discount-am,
+.delivery,
+.total-am,
+.saved-am{
```

Comment:
Instead of having multiple classes using this property create a single class and use it in your HTML for better convenience

Reply:
Good idea. will change it.